sotodlib/tod_ops/gapfill.py

```diff
@@ -380,3 +383,69 @@ def get_contaminated_ranges(good_flags, bad_flags):
                 rs.add_interval(int(i0), int(i1))
     return contam
 
+
+def fill_glitches(aman, nbuf=10, use_pca=False, modes=3, signal=None,
+                  glitch_flags=None, merge_filled=True,
+                  filled_name='gap_filled'):
+    """
+    Top level function for filling glitches assuming glitch flags exist.
+    Wraps the other functions in the ``tod_ops.gapfill`` module.
+
+    Args
+    -----
+    aman : AxisManager
+        AxisManager to fill glitches in
+    nbuf : int
+        Number of buffer samples to use in polynomial gap filling.
+    use_pca : bool
+        Whether or not to fill glitches using pca model. Default is False
+    modes : int
+        Number of modes in the pca to use if pca=True. Default is 3.
+    signal : ndarray or None
+        Array of data to fill glitches in. If None then uses ``aman.signal``.
+        Default is None.
+    glitch_flags : RangesMatrix or None
+        RangesMatrix containing flags to use for gap filling. If None then
+        uses ``aman.flags.glitches``.
+    merge_filled : bool
+        Merge gap filled data into input axis manager.
+    filled_name : str
+        Axis name to merge gap filled data if merge_filled is True.
```

Comment:
In other places we have used "wrap" to support the thing these two arguments are doing.  E.g.:

wrap=False -> no, don't merge it just return it
wrap=True -> yes, merge the new array and call it the default thing ("gap_filled" apparently)
wrap="my_other-name" -> yes, merge the new array and cal it "my_other_name"

Can the preprocess system cope with that setup?  Can you use generic "wrap" instead of specific "merge_filled, filled_name"?